google/cloud/bigtable/examples/README.md

```diff
@@ -70,12 +79,96 @@ offers a simple way to do so:
 $ gcloud auth login
 ```
 
-## Running the Quick Start Examples using a Cloud Bigtable Instance
+## Samples
+
+### Hello world
+
+View the [Hello World][hello_world_code] example to see sample usage of the Bigtable client library.
+
+#### Usage
 
+```console
+$ ./bigtable_hello_world
+Usage: bigtable_hello_world <project_id> <instance_id> <table_id>
+```
+
+#### Run Hello world
 After configuring gRPC, you can run the examples using:
 
 ```console
-$ export PROJECT_ID=... # The name of your project
-$ export INSTANCE_ID=... # THe name of your instance
+$ export PROJECT_ID=... # Your Google Cloud Platform project ID
```

Comment:
See the code here:

https://github.com/GoogleCloudPlatform/google-cloud-cpp/pull/745/files#diff-0b42aa6f16cbd28c471e1b79b6315036R75

particularly around the documentation for these environment variables.